README.md

```diff
@@ -1479,31 +1503,34 @@ less than a year from purchase, but profits from these sales are taxed as ordina
 For a lot of people who make most of their money by working, ordinary income tax is the
 biggest chunk of tax they pay.
 
-- 🄳 **Employment taxes** are an additional kind of federal tax beyond ordinary income tax,
-  and consist of Social Security and
-  [Medicare taxes](https://www.irs.gov/businesses/small-businesses-self-employed/questions-and-answers-for-the-additional-medicare-tax)
-  that are withheld from your paycheck.
-  The Social Security wage withholding rate is **6.2%** up to the FICA wage base.
-  The Medicare component is **1.45%**, and it does not phase out above the FICA wage base.
-  You’ll also hear these called **[payroll taxes](https://en.wikipedia.org/wiki/Payroll_tax)** as
-  they often show up on your pay stub.
+🄳 **Employment taxes** are an additional kind of federal tax beyond ordinary income tax,
+and consist of Social Security and
+[Medicare taxes](https://www.irs.gov/businesses/small-businesses-self-employed/questions-and-answers-for-the-additional-medicare-tax)
+that are withheld from your paycheck.
+The Social Security wage withholding rate is **6.2%** up to the FICA wage base.
+The Medicare component is **1.45%**, and it does not phase out above the FICA wage base.
+You’ll also hear these called **[payroll taxes](https://en.wikipedia.org/wiki/Payroll_tax)** as
+they often show up on your pay stub.
+
 - 🚧 Review and add more links on SS and Medicare taxes.
-- 🄳 **Long-term capital gains tax** is a tax on the sale of assets held longer than a year.
-  Long-term capital gains tax is often lower than ordinary income tax.
-  Many investors hold assets for longer than a year in order to qualify for the lesser tax
-  burden of long-term capital gains.
-- 🄳 **Alternative Minimum Tax (AMT)** is an
-  [entirely separate kind of tax](https://en.wikipedia.org/wiki/Alternative_minimum_tax) that is
-  neither ordinary income tax nor employment tax, and has its own rules and only applies in
-  some situations.
-  This type of tax does not come up for many people, but higher income earners and people in
-  special situations often have to pay very large AMT bills.
-- 🚧 What is the history and motivation of AMT?
-
-- ❗ AMT is relevant to you if you’re reading this.
-  It’s important to understand because exercising ISOs can trigger AMT. In some cases a *lot*
-  of AMT, *even when you haven’t sold the stock* and have no money to pay.
-  We discuss this [later](#taxes-on-stock-and-options).
+
+🄳 **Long-term capital gains tax** is a tax on the sale of assets held longer than a year.
+Long-term capital gains tax is often lower than ordinary income tax.
+Many investors hold assets for longer than a year in order to qualify for the lesser tax
+burden of long-term capital gains.
+🄳 **Alternative Minimum Tax (AMT)** is an
```

Comment:
Sill consider whether you'd like to combine this with the details on AMT risks in "A bit on how all this fits together" section... (See INTERNAL notes in that section.)